test/unit/specs/App.spec.js

```diff
@@ -72,4 +71,95 @@ describe(`App Start`, () => {
     expect(startApp).toHaveBeenCalled()
     expect(startApp.mock.calls[0][0]).toHaveProperty(`x`, 1)
   })
+
+  it(`Check the calls on VUE`, async () => {
```

Comment:
I would have maybe just called the function and checked if it doesn't throw.

Reply:
yeah I tried to test the real call but was taking a weird path to mock them, so I just counted the number of calls